### PR TITLE
Bookpage

### PR DESCRIPTION
**Pull request by divyalakshmi1011**
Hi mam, done with bookpage task. Please check mam.

### CODE REVIEW

File: Project1/application.py [hunk 1]
```diff
@@ -62,8 +62,8 @@ def authenticate() :
         if user :
             salt = user.password[:64]
             stored_password = user.password[64:]
-            # pwdhash = hashlib.pbkdf2_hmac('sha512', password.encode('utf-8'), salt.encode('ascii'), 100000)
-            # pwdhash = binascii.hexlify(pwdhash).decode('ascii')
+            pwdhash = hashlib.pbkdf2_hmac('sha512', password.encode('utf-8'), salt.encode('ascii'), 100000)
+            pwdhash = binascii.hexlify(pwdhash).decode('ascii')
```
Comment (manasa133) [inline Project1/application.py]: use meaningfull variable names

File: Project1/application.py [hunk 2]
```diff
@@ -110,5 +110,29 @@ def admin() :
     users = User.query.order_by(User.timestamp.desc()).all()
     return render_template("admin.html", users=users)
 
+@app.route("/bookpage")
+def bookpage(isbn):
+    # Get data about book from database
+        result = db.execute("SELECT isbn, title, author, year FROM books WHERE id = :id",
```
Comment (manasa133) [inline Project1/application.py]: As sql queries change for different databases, use db.query

Comment (divyalakshmi1011) [reply Project1/application.py]: Mam, please check "application-bookpage" commit. You have reviewed the outdated one. I have updated the changes as  "application-bookpage" commit.Please review it. I have used db.query only.

Comment (manasa133) [reply Project1/application.py]: ok

**Comment by manasa133** [comment]
Merge with Master branch

**Comment by divyalakshmi1011** [comment]
Merged mam.

File: Project1/application.py (hunk 1 above)
Comment (Himaja63) [reply Project1/application.py]: ok ma'am

Comment (Himaja63) [reply Project1/application.py]: can I merge ma'am

Comment (Himaja63) [reply Project1/application.py]: Ma'am pwdhash means changing password to hash. It is taken from divya's code base. If I change now merge conflicts will arise

**Pull request merged**